src/components/Launcher.js

```diff
@@ -47,7 +47,7 @@ class Launcher extends Component {
     ];
     return (
       <div id="sc-launcher">
-        <div className={classList.join(' ')} onClick={this.handleClick.bind(this)}>
+        <div className={classList.join(' ')} onClick={this.handleClick.bind(this)} style={{background: `${this.props.launcherColor}`}}>
```

Comment:
It would be preferable to do this via the `classList` - inline styles can create confusion down the line.